sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/utils/CalciteUtils.java

```diff
@@ -49,6 +49,7 @@
           .put(FieldType.DECIMAL, SqlTypeName.DECIMAL)
           .put(FieldType.BOOLEAN, SqlTypeName.BOOLEAN)
           .put(FieldType.BYTES, SqlTypeName.VARBINARY)
+          .put(FieldType.BYTES, SqlTypeName.BINARY)
```

Comment:
I am not getting this change. After it, it will have `BYTES` mapping to both `BINARY` and `VARBINARY`. It does not pass compilation.

Reply:
There is no support for BINARY in `BeamSqlPrimitive.java1

Reply:
In SQL itself, `BINARY` and `VARBINARY` are not distinguished significantly. I prefer to have both mapping to `BYTES`, same as `CHAR` and `VARCHAR`.